src/Makefile

```diff
@@ -278,6 +274,14 @@ ifneq ($(OS), WINNT)
 	@ln -sf libjulia-debug.$(JL_MAJOR_MINOR_SHLIB_EXT) $(build_shlibdir)/libjulia-debug.$(JL_MAJOR_SHLIB_EXT)
 	@ln -sf libjulia-debug.$(JL_MAJOR_MINOR_SHLIB_EXT) $(build_shlibdir)/libjulia-debug.$(SHLIB_EXT)
 endif
+ifeq ($(USE_POLLY_ACC), 1)
+# Place libGPURuntime to be accessible under libjulia's RPATH
+ifeq ($(OS), WINNT)
+	@cp $(shell $(LLVM_CONFIG_HOST) --libdir)/libGPURuntime.dll $(build_shlibdir)
+else
+	@cp $(shell $(LLVM_CONFIG_HOST) --libdir)/libGPURuntime.$(SHLIB_EXT) $(build_shlibdir)
```

Comment:
`grep SHLIB_EXT= -Rsn` gave me this,
```
contrib/fixup-libgfortran.sh:21:    SHLIB_EXT="so"
contrib/fixup-libgfortran.sh:23:    SHLIB_EXT="dylib"
contrib/fixup-libgfortran.sh:27:    SHLIB_EXT="so"
```
So, I thought I had to handle the case for Windows. Anyways, I doubt that fixup-libgfortran.sh is setting the SHLIB_EXT used in src/Makefile. How is it getting set then ?